src/Core/Utilities/Plan/EnterprisePlan.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Bit.Core.Enums;
+
+namespace Bit.Core.Utilities.Plan;
+
+public record EnterprisePlan : Models.StaticStore.Plan
+{
+    public EnterprisePlan(bool isAnnual)
```

Comment:
Please break these property assignments up into blocks (that is, add some line breaks between sections), similar to how `PasswordManagerPlanStore` is currently. It makes it much more readable.